api/src/main/java/org/openmrs/ConditionClinicalStatus.java

```diff
@@ -39,5 +39,8 @@
 	 * Remission is where the patient is no longer experiencing the symptoms of the condition, 
 	 * but there is a risk of the symptoms returning.
 	 * */
-	HISTORY_OF 
+	HISTORY_OF
+	/**
+	 *  just a test comment
```

Comment:
probably some works are still coming in for this @abertnamanya

Reply:
Yeah , I was testing if I revert would it still show the errors,